src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -3604,6 +3612,10 @@ def _handle_addons_args(cmd, addons_str, subscription_id, resource_group_name, a
             addon_profile.config[CONST_ACC_SGX_QUOTE_HELPER_ENABLED] = "true"
         addon_profiles[CONST_CONFCOM_ADDON_NAME] = addon_profile
         addons.remove('confcom')
+    if 'open-service-mesh' in addons:
+        addon_profile = ManagedClusterAddonProfile(enabled=True, config={})
+        addon_profiles[CONST_OPEN_SERVICE_MESH_ADDON_NAME] = addon_profile
+        addons.remove('open-service-mesh')
```

Comment:
Hi, we have refactored the implementation of `aks create` and function `_handle_addons_args` would not be used in `aks create` anymore. Could you please add this part to this [file](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/decorator.py#L3712)?

Reply:
@FumingZhang, resolved this.